modules/installation-aws-limits.adoc

```diff
@@ -69,7 +69,7 @@ To use the `us-east-1` region, you must increase the EIP limit for your account.
 |3
 |20 per region
 |By default, each cluster creates internal and external network load balancers for the master
-API server and a single classic elastic load balancer for the router. Deploying
+API server and a single classic ELB load balancer for the router. Deploying
```

Comment:
@dfitzmau The heading above intros the service name, so there's no need to repeat here. Suggested revision: "single Classic Load Balancer for the router."